ThunderCloud/JSONString.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  JSONString.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 03/04/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+extension String {
```

Comment:
Useful! :D

Reply:
Going to leave as-is for now for time constraints :) Not sure about having a `throw` on `init` either, seems a bit strange!

Reply:
Sure :)! Btw I think `throws` in an `init` is fine when multiple errors can occur and nullability isn't enough :P

Reply:
Ahh yeah that's a fair point!